docs/reference_behavior_attributes.md

```diff
@@ -105,6 +105,16 @@ These elements support the `load` trigger:
 - [`<text>`](/docs/reference_text)
 - [`<image>`](/docs/reference_image)
 
+### `unload`
+
+Triggers when the element is unloaded in the screen.
+
+> Elements with `hide="true"` are not unloaded in the screen. This means any hidden element (or child element) with the `unload` trigger will not be executed.
```

Comment:
Maybe to make this clear, we should say "Elements with hide=true are never loaded. Thus, they are not unloaded and will not trigger an `unload` behavior."